common/lib/xmodule/xmodule/tests/test_poll.py

```diff
@@ -30,3 +33,29 @@ def test_good_ajax_request(self):
         self.assertEqual(total, 2)
         self.assertDictEqual(callback, {'objectName': 'Conditional'})
         self.assertEqual(self.xmodule.poll_answer, 'No')
+
+    def test_poll_export_with_unescaped_characters_xml(self):
+        """
+        Make sure that poll_module will export fine if its xml contains
+        unescaped_characters.
```

Comment:
nit: remove underscore. Change `unescaped_characters.` to `unescaped characters.`